test/windows_test.go

```diff
@@ -682,3 +690,35 @@ func testWindowsDefaultPlatform(ctx context.Context, t *testing.T) {
 		solveT(ctx, t, gwc, sr)
 	})
 }
+
+func testWindowsZipFilename(ctx context.Context, t *testing.T) {
+	spec := newSimpleSpec()
+	spec.Build.Steps = []dalec.BuildStep{
+		{
+			Command: "echo \"hello world\" >> output.ps1",
+		},
+	}
+	spec.Artifacts.Binaries = map[string]dalec.ArtifactConfig{
+		"output.ps1": {},
+	}
+
+	testEnv.RunTest(ctx, t, func(ctx context.Context, gwc gwclient.Client) {
+		sr := newSolveRequest(withSpec(ctx, t, spec), withBuildTarget("windowscross/zip"))
+		res := solveT(ctx, t, gwc, sr)
+		ref, err := res.SingleRef()
+		if err != nil {
+			t.Fatalf("shouldn't get an error wile calling SingleRef()")
+		}
+		filename := fmt.Sprintf("/%s_%s-%s_%s.zip", spec.Name, spec.Version, spec.Revision, runtime.GOARCH)
+		stat, err := ref.StatFile(ctx, gwclient.StatRequest{Path: filename})
+		if err != nil {
+			t.Fatalf("should have gotten a file: %s: %s", filename, err)
+		}
```

Comment:
```suggestion
		assert.NilError(t, err)
```

Reply:
applied, forgot to swap to assert when I pushed this (we discussed this while pairing 😢 )